### PR TITLE
re-instate npm audit

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves N/A

**Overall change:** Add back the audit-ci package used to run npm on on CI.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [N/A] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by DemonusPC — APPROVED** [review]
Lets just hope that this will not `rip in pepperamis` for us this time

**Review by mbzain76 — APPROVED** [review]
Ok

**Comment by chris-hinds** [comment]
NPM State this service is stable again [NPM Issue Tracker](https://status.npmjs.org/incidents/1jdmllbx0q59?_ga=2.107255874.1130125350.1558356462-877631025.1558356462)

File: Makefile [hunk 1]
```diff
@@ -10,6 +10,7 @@ installProd:
 	npm ci --only=production
 
 developmentTests:
+	npm run audit:ci
```
Comment (sareh) [inline Makefile]: Can this please be run after the other tests? It's a better dev experience if npm audit becomes flaky again and times out, to have the unit tests and storybook e2e tests passing first.

Comment (chris-hinds) [reply Makefile]: good idea.

**Comment by PriyaKR** [comment]
This PR wouldn't require any testing from a tester.

**Comment by chris-hinds** [comment]
@sareh are you happy with the changes?

File: .travis.yml [hunk 2]
```diff
@@ -24,20 +24,21 @@ jobs:
         - npm run test
         - ./cc-test-reporter after-build -t lcov --debug --exit-code $TRAVIS_TEST_RESULT
         - xvfb-run -a npm run test:e2e:storybook:ci
+        - npm run audit:ci
```
Comment (jamesbhobbs) [inline .travis.yml]: why run on the dev install? this will fail for non prod env vulns

**Review by sareh — APPROVED** [review]
👍 Thanks for the updates!

**Pull request merged**